brianlong/www.validators.app
# 186502853 bring back logos refresh

#### What's this PR do?
- DESCRIBE THE PURPOSE OF THE PR HERE

#### How should this be manually tested?
On staging:
- let the script run, or fire it manually, and check the following validator: https://stage.validators.app/validators/XkCriyrNwS3G4rzAXtG5B1nnvb5Ka1JtCku93VqeKAr?locale=en&network=mainnet
- The old logo should be updated to this one: https://s3.amazonaws.com/keybase_processed_uploads/68d8ed237f31e993a8eac1bca7512a05_360_360.jpg

On localhost:
- set logo to an old one: `Validator.find_by(account: "XkCriyrNwS3G4rzAXtG5B1nnvb5Ka1JtCku93VqeKAr").update(avatar_url: "https://s3.amazonaws.com/keybase_processed_uploads/a1413845d3aa5e3fec212768c3a87c05_360_360.jpg")`
- run update script: `rails r script/validators_update_avatar_url.rb`
- confirm that Coinbase's logo is now updated http://localhost:3000/validators/XkCriyrNwS3G4rzAXtG5B1nnvb5Ka1JtCku93VqeKAr?locale=en&network=mainnet

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186502853)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
